getting-started/default_posture.md

```diff
@@ -0,0 +1,149 @@
+# Default Posture
+
+KubeArmor supports configurable default mode of operations. Default Posture comes into picture when there's atleast one `Allow` policy i.e. KubeArmor is handling policies in whitelisting manner ( More about this in [Consideration in Policy Action](https://github.com/kubearmor/KubeArmor/blob/event-auditor/getting-started/consideration_in_policy_action.md) ).
+
+There are two default mode of operations available `block` and `audit`. `block` mode blocks all the operations that are not allowed in the policy. `audit` generates telemetry events for operations that would have been blocked otherwise.
+
+KubeArmor has 4 types of resources: Process, File, Network and Capabilities. Default Posture is configurable for each of the resources seperately except Process. Process based operations are treated under File resource only.
+## Configuring Default Posture
+
+### Global Default Posture
+
+Global default posture is configured using configuration options provided in KubeArmor binary.
+```sh
+  -defaultFilePosture string
+    	configuring default enforcement action in global file context [audit,block] (default "block")
+  -defaultNetworkPosture string
+    	configuring default enforcement action in global network context [audit,block] (default "block")
+  -defaultCapabilitiesPosture string
+    	configuring default enforcement action in global capability context [audit,block] (default "block")
+```
+
+### Namespace Default Posture
+
+We use namespace annotations to configure default posture per namespace. Supported annotations keys are `kubearmor-file-posture`,`kubearmor-network-posture` and `kubearmor-capabilities-posture` with values `block` or `audit`. If a namespace is annotated with a supported key and an invalid value ( like `kubearmor-file-posture=invalid`), KubeArmor will update the value with the global default posture ( i.e. to `kubearmor-file-posture=block`).
+
```

Comment:
It would be better to give an example using a sample yaml with annotation.

Reply:
On updating annotations using YAML 
```yaml
apiVersion: v1
kind: Namespace
metadata:
  name: multiubuntu
  annotations:
    kubearmor-network-posture: audit
```
`kubectl apply -f ...`
If we only update metadata (annotation), it prints `unchanged` despite updating the annotation value anyway. So it feels more right to use `kubectl annotate` since it responds accordingly.

WDYT?

Reply:
'kubectl annotate' would be a good choice.